src/server/lib/__tests__/okta/api/sessions.test.ts

```diff
@@ -0,0 +1,66 @@
+import { mocked } from 'jest-mock';
+import { fetch } from '@/server/lib/fetch';
+import type { Response, RequestInfo, RequestInit } from 'node-fetch';
+import { getSession } from '@/server/lib/okta/api/sessions';
+import { OktaError } from '@/server/models/okta/Error';
+
+const sessionId = '123';
+
+// mocked configuration
+jest.mock('@/server/lib/getConfiguration', () => ({
+  getConfiguration: () => ({
+    okta: {
+      enabled: true,
+      orgUrl: 'someOrgUrl',
+      token: 'token',
+      authServerId: 'authServerId',
+      clientId: 'clientId',
+      clientSecret: 'clientSecret',
+    },
+  }),
+}));
+
+// mocked fetch
+jest.mock('@/server/lib/fetch');
+const mockedFetch =
+  mocked<(url: RequestInfo, init?: RequestInit) => Partial<Promise<Response>>>(
+    fetch,
+  );
```

Comment:
I feel I should mention that I tried [MSW](https://mswjs.io) and it's the nicest request mocking solution I've ever used. Has the nice benefit of working across browsers and node, so you can re-use the same mocks in cypress and in jsdom tests if you want to.